### DISCUSSION TITLE
Create area delete button

### DISCUSSION THREAD

**Discussion by mariomatasystems** [cables-gl/cables_docs#844] [Feature Requests]
Sometimes after creating an area around a bunch of ops you'd like to delete the area, could be nice to have a little cross somewhere, maybe top right where you can delete the area without deleting all the ops inside of it.

Currently i find myself either reducing the area with the arrow bottom right so it doesn't cover any ops and then pressing delete, or I find myself moving the ops out of the area, then deleting the area, then moving my ops back in place.

**Comment by wayfu** [comment]
alt-LMB-dragging the area by its registration will move it without moving the ops inside it, that's a nicer workaround at least for the moment

**Reply by mariomatasystems** [reply]
> alt-LMB-dragging the area by its registration will move it without moving the ops inside it, that's a nicer workaround at least for the moment

This is GREAT to know
